### PR TITLE
remove auth in governance

### PR DESCRIPTION
**Pull request by siovanus**

### CODE REVIEW

File: smartcontract/service/native/governance/governance.go [hunk 1]
```diff
@@ -95,10 +95,11 @@ const (
...
 	//global
-	PRECISE            = 1000000
-	NEW_VERSION_VIEW   = 6
-	NEW_VERSION_BLOCK  = 414100
-	NEW_WITHDRAW_BLOCK = 2800000
+	PRECISE                = 1000000
+	NEW_VERSION_VIEW       = 6
+	NEW_VERSION_BLOCK      = 414100
+	NEW_WITHDRAW_BLOCK     = 2800000
+	NEW_DECENTRALIZE_BLOCK = 7000000
```
Comment (Honglei-Cong) [inline smartcontract/service/native/governance/governance.go]: suggest renaming as "BLOCKHEIGHT_SELFGOV_REGISTER" ?

Comment (siovanus) [reply smartcontract/service/native/governance/governance.go]: Done

**Review by Honglei-Cong — DISMISSED** [review]

File: smartcontract/service/native/governance/governance.go [hunk 2]
```diff
@@ -95,10 +95,11 @@ const (
...
 	//global
-	PRECISE            = 1000000
-	NEW_VERSION_VIEW   = 6
-	NEW_VERSION_BLOCK  = 414100
-	NEW_WITHDRAW_BLOCK = 2800000
+	PRECISE                      = 1000000
+	NEW_VERSION_VIEW             = 6
+	NEW_VERSION_BLOCK            = 414100
+	NEW_WITHDRAW_BLOCK           = 2800000
+	BLOCKHEIGHT_SELFGOV_REGISTER = 7000000
```
Comment (laizy) [inline smartcontract/service/native/governance/governance.go]: distinguish mainnet and testnet? and the height of  mainnet is already `8343179 `.

File: smartcontract/service/native/governance/method.go [hunk 3]
```diff
@@ -97,18 +99,89 @@ func registerCandidate(native *native.NativeService, flag string) error {
...
+		//check if has index
+		peerPubkeyPrefix, err := hex.DecodeString(peerPoolItem.PeerPubkey)
+		if err != nil {
+			return fmt.Errorf("registerCandidate, peerPubkey format error: %v", err)
+		}
+		indexBytes, err := native.CacheDB.Get(utils.ConcatKey(contract, []byte(PEER_INDEX), peerPubkeyPrefix))
+		if err != nil {
+			return fmt.Errorf("registerCandidate, get indexBytes error: %v", err)
+		}
+		if indexBytes != nil {
```
Comment (laizy) [inline smartcontract/service/native/governance/method.go]: use `len(indexBytes) != 0`

**Review by laizy — DISMISSED** [review]

**Comment by laizy** [comment]
resolve confilicts @siovanus

**Comment by siovanus** [comment]
> resolve confilicts @siovanus

Done

**Review by laizy — APPROVED** [review]

**Pull request merged**